caching-service/src/main/resources/application.yml

```diff
@@ -41,6 +41,7 @@ logging:
         org.apache.tomcat.util.net.SSLUtilBase: ERROR
         org.springframework.security.config.annotation.web.builders.WebSecurity: ERROR
         javax.net.ssl: ERROR
+    timezone: ${apiml.logging.timezone:UTC}
```

Comment:
Please describe this field in the additional-spring-configuration-metadata.json. Ideally with sample values.